state/logs.go

```diff
@@ -231,7 +229,6 @@ func (logger *DbLogger) Log(t time.Time, entity string, module string, location
 		Time:      unixEpochNanoUTC,
```

Comment:
Please add a TODO near `logDoc`'s definition which references the ticket you created about removing `Version` from the struct.